torch/csrc/jit/frontend/tracer.h

```diff
@@ -179,7 +179,7 @@ inline void warn(const char* _reason, const char* _kind = nullptr) {
 TORCH_API void setWarn(warn_fn_type fn);
 
 struct TORCH_API NoWarn {
-  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
+  // NOLINTNEXTLINE(*-member-init, *.cplusplus.UninitializedObject)
```

Comment:
Hmm, think this technically needs to be on the next line. Seems like the parser doesn't recognize state is defined as a member initializer, which is quite odd. Maybe because it's initialized by a method?